docs/modules/ROOT/content-nav.adoc

```diff
@@ -5,4 +5,5 @@
 ** xref:getting-started/filters-and-projections.adoc[]
```

Comment:
I just realized now that the Getting started section is using a wrong format for capitalizing the title, it should be "Getting started" instead of "Getting Started", but I cannot make the suggestion here 😅

Reply:
Good catch, I'll make the change